repose-aggregator/core/repose-core/src/main/java/org/openrepose/nodeservice/response/LocationHeaderBuilder.java

```diff
@@ -56,6 +60,24 @@ public static void setLocationHeader(HttpServletRequest originalRequest, HttpSer
         }
     }
 
+    private static URL getProxiedHostUrl(String targetHost) {
+        URL returnUrl = null;
+        URI targetUri = null;
+        try {
+            targetUri = new URI(targetHost);
+        } catch (URISyntaxException e) {
+            LOG.error("Invalid target host url: " + targetHost, e);
```

Comment:
This should be a `WARN` at most since processing is able to continue.

Reply:
This class is actually slated for deletion. In fact, the method where this message would be logged is no longer in use. That being the case, I'll do you one better and delete this dead code.

On it.